support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -413,46 +428,57 @@ export function ThreeTierLanding({
 	 *
 	 * This should only exist as long as the Tier three hack is in place.
 	 */
-	const getTier3RatePlan = () => {
-		const ratePlanKey =
-			countryGroupId === 'International'
-				? contributionType === 'ANNUAL'
-					? 'RestOfWorldAnnual'
-					: 'RestOfWorldMonthly'
-				: contributionType === 'ANNUAL'
-				? 'DomesticAnnual'
-				: 'DomesticMonthly';
-
-		return ratePlanKey;
-	};
-
-	const tier3RatePlan = getTier3RatePlan();
-	const tier3Pricing = productCatalog.TierThree?.ratePlans[tier3RatePlan]
+	const tier3Product = enablePremiumDigital
+		? 'DigitalSubscription'
+		: 'TierThree';
+	const tier3RatePlanKey = enablePremiumDigital
+		? ratePlanKey
+		: getThreeTierRatePlanKey(contributionType, countryGroupId);
+	const tier3Pricing = productCatalog[tier3Product]?.ratePlans[tier3RatePlanKey]
 		?.pricing[currencyId] as number;
-
 	const tier3UrlParams = new URLSearchParams({
-		product: 'TierThree',
-		ratePlan: tier3RatePlan,
+		product: tier3Product,
+		ratePlan: tier3RatePlanKey,
 	});
-	const tier3Promotion = getPromotion(
-		allProductPrices.TierThree,
-		countryId,
-		billingPeriod,
-		countryGroupId === 'International' ? 'RestOfWorld' : 'Domestic',
-		enablePremiumDigital ? 'NewspaperArchive' : 'NoProductOptions',
-	);
+	const premiumDigitalProductDescription = {
+		title: 'Premium digital',
+		benefits: filterProductDescriptionBenefits(
+			productCatalogDescriptionPremiumDigital,
+			countryGroupId,
+		),
+		cta: {
+			copy: 'Subscribe',
+		},
+	};
+	const tier3ProductDescription = enablePremiumDigital
+		? premiumDigitalProductDescription
+		: settings.products.TierThree;
+	const tier3ProductPrice = enablePremiumDigital
+		? allProductPrices.DigitalPack
+		: allProductPrices.TierThree;
+	const tier3Promotion = tier3ProductPrice
+		? getPromotion(
+				tier3ProductPrice,
+				countryId,
+				billingPeriod,
+				countryGroupId === 'International' ? 'RestOfWorld' : 'Domestic',
+				enablePremiumDigital ? 'NewspaperArchive' : 'NoProductOptions',
+		  )
+		: undefined;
 	if (tier3Promotion) {
 		tier3UrlParams.set('promoCode', tier3Promotion.promoCode);
 	}
+	const featureFlag = enablePremiumDigital ? `enablePremiumDigital&` : '';
 	const tier3Card: CardContent = {
-		product: 'TierThree',
+		product: tier3Product,
 		price: tier3Pricing,
-		link: `checkout?${tier3UrlParams.toString()}`,
+		link: `checkout?${featureFlag}${tier3UrlParams.toString()}`,
```

Comment:
If `featureFlag` was conditionally added to the `tier3UrlParams` above then we could just let the `URLSearchParams.toString()` handle getting the query string correct. Like we do for `promoCode`.